salt/addons/deploy_addons.sh

```diff
@@ -26,9 +26,19 @@ fi
 
 echo "Deploying DNS on Kubernetes"
 
+KUBEDNS_SA=`eval "$KUBECTL_PATH $NS_ARGS $SRV_ARGS get sa | grep kube-dns | cat"`
```

Comment:
That's outside of the scope of this change. A replacement of this script is in progress.